badook_tests/context/context.py

```diff
@@ -35,12 +35,12 @@ def collect(self, data, path=''):
             raise Exception(
                 f"collect summaries - {response['error']['message']}")
 
-    def create_summary(self, name, dataset) -> DatasetSummary:
+    def get_dataset(self, dataset) -> DatasetSummary:
```

Comment:
Maybe rename as follows?
```suggestion
    def get_dataset_summary(self, dataset) -> DatasetSummary:
```

Reply:
because we are moving away from Summaries to Metrics as a terminology I don't think it makes sense to keep it here